### ISSUE TITLE
Design review

### ISSUE THREAD

**Issue by Moeriki** [OpenSkool/OpenSkool#236]

**Comment by pbuytaert** [comment]
The active state of the switch button is to dark. It should be the main yellow

**Comment by pbuytaert** [comment]
Not sure it should go here, but we don't of an H1 on the page.

**Comment by pbuytaert** [comment]
Remark: we will always use the **line** version of an icon. The wireframes contain the name of the correct icon

**Comment by Moeriki** [comment]
Yes I noticed. For me it's part of #208 which has yet to be done. I'll update ticket to make it explicit.
